dpnp/backend/include/dpnp_iface.hpp

```diff
@@ -2173,111 +1298,6 @@ INP_DLLEXPORT void dpnp_repeat_c(const void *array_in,
                                  const size_t repeats,
                                  const size_t size);
 
-/**
- * @ingroup BACKEND_API
- * @brief transpose function. Permute axes of the input to the output with
- * elements permutation.
- *
- * @param [in]  q_ref               Reference to SYCL queue.
- * @param [in]  array1_in           Input array.
- * @param [in]  input_shape         Input shape.
- * @param [in]  result_shape        Output shape.
- * @param [in]  permute_axes        Order of axis by it's id as it should be
- * presented in output.
- * @param [in]  ndim                Number of elements in shapes and axes.
- * @param [out] result1             Output array.
- * @param [in]  size                Number of elements in input arrays.
- * @param [in]  dep_event_vec_ref   Reference to vector of SYCL events.
- */
-template <typename _DataType>
-INP_DLLEXPORT DPCTLSyclEventRef
-    dpnp_elemwise_transpose_c(DPCTLSyclQueueRef q_ref,
```

Comment:
Do we need to remove `dpnp_elemwise_transpose_c` from `dpnp/backend/kernels/dpnp_krnl_manipulation.cpp`?